search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/VariableFilterTransformer.java

```diff
@@ -83,7 +83,10 @@ private SearchQuery getTenantIdQuery(final List<String> tenant) {
     return stringTerms("tenantId", tenant);
   }
 
-  private SearchQuery getIsTruncatedQuery(final boolean isTruncated) {
+  private SearchQuery getIsTruncatedQuery(final Boolean isTruncated) {
+    if (isTruncated == null) {
+      return null;
+    }
     return term("isPreview", isTruncated);
```

Comment:
Does the `isPreview` match the ES/OS field property? If so just giving you a reminder that it will be changed to `isTruncated` for consistency when the schema is moved over the exporter :smiling_face_with_tear:

Reply:
It is :) For now the mapping is still isPreview, so for avoid breaking changes, kets keep it as is, and change when the exporters are released and indexes updates

Thanks for raise this point @panagiotisgts